sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/tests/BlobTriggerTests.cs

```diff
@@ -148,6 +149,41 @@ private class BindToCloudBlob2Program
             }
         }
 
+        [Test]
+        public async Task BlobTrigger_BindingToParameterBindingData()
+        {
+            // Arrange
+            var container = CreateContainer(blobServiceClient, ContainerName);
+            var blob = container.GetBlobClient(BlobName);
+
+            await blob.UploadAsync(BinaryData.FromString("hello world"));
+
+            // Act
+            ParameterBindingData result = await RunTriggerAsync<ParameterBindingData>(typeof(BlobTriggerParameterBindingData),
+                (s) => BlobTriggerParameterBindingData.TaskSource = s);
+
+            var blobData = result.Content.ToObjectFromJson<Dictionary<string,string>>();
+
+            // Assert
+            Assert.True(blobData.TryGetValue("Connection", out var resultConnection));
+            Assert.True(blobData.TryGetValue("ContainerName", out var resultContainerName));
+            Assert.True(blobData.TryGetValue("BlobName", out var resultBlobName));
+
+            Assert.True(resultConnection.Contains(ConnectionName));
```

Comment:
Why are you using contains here? We want to test equality

Reply:
Done